modules/examples/sagemaker/sagemaker-example/lambda_functions/process_mdb_change_event/app.py

```diff
@@ -0,0 +1,84 @@
+import boto3
+import json
```

Comment:
let's remove the sagemake example code from this repo and link to the code in the cloud-formation for this https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration

Reply:
Removed